h/api/presenters.py

```diff
@@ -28,6 +38,28 @@ def links(self):
                 out[name] = link
         return out
 
+    @property
+    def text(self):
+        if self.annotation.text:
+            return self.annotation.text
+        else:
+            return ''
+
+    @property
+    def tags(self):
+        if self.annotation.tags:
+            return self.annotation.tags
+        else:
+            return []
+
+    @property
+    def target(self):
+        target = {'source': self.annotation.target_uri}
+        if self.annotation.target_selectors:
+            target['selector'] = self.annotation.target_selectors
```

Comment:
I'll note here that the format of some of the selectors, eg. the RangeSelector in the spec ( https://hyp.is/AVPw1LheH9ZO4OKSlnLY/www.w3.org/TR/annotation-model/ ) differs from the type of the selector with the same name in our existing annotations, so they would need to be transformed as well to be fully compliant.